go/worker/event/kafkaServer.go

```diff
@@ -24,32 +25,92 @@ type KafkaClient interface {
 	Close()
 }
 
-// LambdaKafkaConsumer manages Kafka consumption for a specific lambda function
+// TriggerKey uniquely identifies a Kafka trigger configuration
+type TriggerKey string
+
+// ComputeTriggerKey creates a deterministic key from trigger config
+func ComputeTriggerKey(trigger *common.KafkaTrigger) TriggerKey {
+	// Sort brokers and topics for consistency
+	brokers := make([]string, len(trigger.BootstrapServers))
+	copy(brokers, trigger.BootstrapServers)
+	sort.Strings(brokers)
+
+	topics := make([]string, len(trigger.Topics))
+	copy(topics, trigger.Topics)
+	sort.Strings(topics)
+
+	// Create canonical key: brokers|topics|offset_reset
```

Comment:
Ok to make it a TODO for now.  But do we need to escape this?  Could "|" be in the topic name?